metric_wavefront.go

```diff
@@ -29,6 +37,34 @@ func initMetricWavefront() error {
 	return nil
 }
 
+func extractWavefrontUrl(wavefrontUrl string) string {
```

Comment:
Why not just expect the whole URL to be set in the env, in a single variable, and get rid of this method altogether?

Reply:
Because kubernetes :(

Reply:
I prefer to clearly write the environment variables names in the configuration. If at some point the name of the variables in Kubernetes change, we only need to change them in the configuration file, and not to change the hardcoded value from `owl`, push a new version, import the new version in `log-processing-go`, creating a new Docker container, etc.

TL;DR If Kubernetes changes the environment variable naming, it's faster to change it in a configuration file.

Reply:
Ok, forget about the env, but why do we need to parse something that we already provide in the conf?